vllm/model_executor/layers/sampler.py

```diff
@@ -47,6 +47,28 @@ def __init__(self):
         # speculative decoding.
         self.include_gpu_probs_tensor = False
 
+    def _init_sampling_tensors(
```

Comment:
docstring when this is used // what the intention is ?

Reply:
added